client/src/components/layouts/GuideLayout.js

```diff
@@ -68,8 +68,8 @@ class Layout extends React.Component {
                 toggleExpandedState
               }) => (
                 <Fragment>
-                  <Spacer size={2} />
-                  <Grid>
+                  <Spacer size={1} />
+                  <Grid fluid={true}>
```

Comment:
setting fluid to true is spreading the grid to the the screeen's edges. I was wondering if you are getting similar results on your side. 

<img width="1383" alt="screen shot 2019-01-31 at 3 07 22 pm" src="https://user-images.githubusercontent.com/4591597/52060898-45488b80-257e-11e9-8d00-072d04ec077a.png">



The following is what I am getting for the current master.
<img width="1386" alt="screen shot 2019-01-31 at 3 04 53 pm" src="https://user-images.githubusercontent.com/4591597/52061026-817bec00-257e-11e9-92fb-c31371b94352.png">

Reply:
You're right, we do not require this anymore, I think a recent PR has fixed what I was seeing that day